tests/contrib/test_brotli.py

```diff
@@ -0,0 +1,34 @@
+import logging
+from typing import cast
+
+from starlette.types import ASGIApp
+
+from starlite import get
+from starlite.config import BrotliConfig
+from starlite.contrib.brotli import BrotliMiddleware, BrotliMode
+from starlite.testing import create_test_client
+
+logger = logging.getLogger(__name__)
+
+
+@get(path="/")
+def handler() -> None:
+    ...
+
+
+def test_brotli_middleware() -> None:
```

Comment:
So, for the amount of code you are adding this test is really not sufficient. I would expect a more or less full suite of unit tests + some integration tests to ensure this is working properly.

Reply:
No issue.  I agree here.  I'll add additional test cases to properly check for compression headers and validate sizing.